app/src/main/java/com/bootcamp_android/parking_app/fragments/LotListFragment.kt

```diff
@@ -10,7 +12,13 @@ import com.bootcamp_android.parking_app.adapter.Adapter
 import com.bootcamp_android.parking_app.adapter.Provider.Companion.lots
 import com.bootcamp_android.parking_app.databinding.FragmentLotsBinding
 
-class LotListFragment : Fragment(R.layout.fragment_lots) {
+class LotListFragment : Fragment() {
+
+    override fun onCreateView(
```

Comment:
Try to declare the binding with lateinit and clean it in onDestroy
please do the following example https://github.com/android/architecture-components-samples/blob/main/ViewBindingSample/app/src/main/java/com/android/example/viewbindingsample/InflateFragment.kt

The idea is that you can use that binding variable in other methods